src/plugins/shared_ux/public/index.ts

```diff
@@ -5,9 +5,9 @@
  * in compliance with, at your election, the Elastic License 2.0 or the Server
  * Side Public License, v 1.
  */
-
+import React from 'react';
+import { withSuspense } from '@kbn/shared-ux-utility';
```

Comment:
We shouldn't be exporting this.  Consumers should import directly from the package.